spec/services/topology_service_spec.rb

```diff
@@ -143,8 +143,8 @@
 
     context 'entity is a tag' do
       let(:parent_cls) { FactoryBot.create(:classification, :description => 'foo') }
-      let(:cls) { FactoryBot.create(:classification, :parent => parent_cls, :description => 'bar') }
```

Comment:
classification factory has `:parent_id => 0`. Passing in `:parent =>` seems dangerous.
I don't fully trust rails to do the right thing here. so using `classification_tag` (factory without the `:parent` specified` makes more sense.